frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestOverview.tsx

```diff
@@ -306,175 +338,244 @@ export const ChangeRequestOverview: FC = () => {
+                                            Preview changes
+                                        </StyledButton>
+                                    }
+                                />
+
+                                <ConditionallyRender
+                                    condition={
+                                        changeRequest.state !== 'Applied' &&
+                                        changeRequest.state !== 'Rejected' &&
+                                        changeRequest.state !== 'Cancelled' &&
+                                        (changeRequest.createdBy.id ===
+                                            user?.id ||
+                                            isAdmin)
+                                    }
+                                    show={
+                                        <ConditionallyRender
+                                            condition={Boolean(scheduledAt)}
+                                            show={
+                                                <StyledButton
+                                                    variant='outlined'
+                                                    onClick={() =>
+                                                        setShowRejectScheduledDialog(
+                                                            true,
+                                                        )
+                                                    }
+                                                    disabled={disabled}
+                                                >
+                                                    Reject changes
+                                                </StyledButton>
+                                            }
+                                            elseShow={
+                                                <StyledButton
+                                                    variant='outlined'
+                                                    onClick={onCancel}
+                                                    disabled={disabled}
+                                                >
+                                                    Cancel changes
+                                                </StyledButton>
+                                            }
+                                        />
+                                    }
+                                />
+                            </StyledButtonBox>
+                        </StyledInnerContainer>
+                    </StyledPaper>
+                    <ConditionallyRender
+                        condition={
+                            changeRequest.state === 'Approved' &&
+                            approversEnabled
+                        }
+                        show={
+                            <StyledPaper sx={{ mt: 2 }} elevation={0}>
+                                <StyledInnerContainer sx={{ pt: 0, pb: 1.5 }}>
```

Comment:
Shouldn't these properties be added to the Styled component? Or maybe have a different component style?